Gulpfile.js

```diff
@@ -1,16 +1,27 @@
-const gulp = require('gulp');
+const { src, dest, series } = require('gulp');
 const babel = require('gulp-babel');
 const uglify = require('gulp-uglifyjs');
 const rename = require('gulp-rename');
+const clean = require('gulp-clean');
 
-gulp.task('default', () => {
-  return gulp.src('src/index.js')
+function build() {
+  return src('src/index.js', { allowEmpty: true })
```

Comment:
```suggestion
  return src('src/index.js')
```

`allowEmpty` doesn't seem a good option here. We should catch some human errors, one of them maybe is an empty file

Reply:
ok